README.md

```diff
@@ -4,7 +4,6 @@
 
 > Description the project.
```

Comment:
- [ ] Great job so far 👍 Your design looks great. Kindly replace the `Project name` in your README file with the name of your project and provide a brief description of the project so as to make your README file more professional 💯

Reply:
All good now 👍🏽